client/utils/account-fees.tsx

```diff
@@ -86,17 +86,19 @@ export const getCurrentFee = (
 };
 
 export const formatMethodFeesTooltip = (
-	accountFees: FeeStructure | undefined
+	accountFees: FeeStructure
 ): JSX.Element => {
 	if ( ! accountFees ) return <></>;
+	// If there is a discount, use that instead of the base fee.
```

Comment:
We can skip this comment.

Reply:
Sure, I have removed [it here](https://github.com/Automattic/woocommerce-payments/pull/5299/commits/2d8b5afb1599fc98fa4bb1c4651818d20acc4267#diff-3287280562aa1d47c819215cb3d6cfca08ecf177711c1ad791ef2103cf5bf090L92)

Commit - [1ec4acd](https://github.com/Automattic/woocommerce-payments/pull/5299/commits/1ec4acd531489ad3f58d1b99b79a875f498f4862)